tests/modules/seqwish/induce/test.yml

```diff
@@ -1,8 +1,16 @@
-- name: seqwish induce
-  command: nextflow run ./tests/modules/seqwish/induce -entry test_seqwish_induce -c ./tests/config/nextflow.config -c ./tests/modules/seqwish/induce/nextflow.config
+- name: seqwish induce transcriptome
+  command: nextflow run ./tests/modules/seqwish/induce -entry test_seqwish_induce_transcriptome -c ./tests/config/nextflow.config -c ./tests/modules/seqwish/induce/nextflow.config
   tags:
     - seqwish
     - seqwish/induce
   files:
     - path: output/seqwish/test.gfa
       md5sum: 216a02d3aca322a457c31a62c628548e
+
+- name: seqwish induce pangenomce
+  command: nextflow run ./tests/modules/seqwish/induce -entry test_seqwish_induce_pangenome -c ./tests/config/nextflow.config -c ./tests/modules/seqwish/induce/nextflow.config
+  tags:
+    - seqwish
+    - seqwish/induce
+  files:
+    - path: output/seqwish/test.gfa
```

Comment:
What about the md5sum? Can you use a `contains`string in case the number iis not consistent?

Reply:
I added an MD5Sum.